src/Control/Distributed/Closure/Internal.hs

```diff
@@ -167,4 +167,5 @@ cap closf closx = Ap closf closx
 -- over it. That is, we cannot define 'fmap'. However, we can map a static
 -- pointer to a function over a 'Closure'.
 cmap :: Typeable a => StaticPtr (a -> b) -> Closure a -> Closure b
-cmap sptr = cap (Closure (deRefStaticPtr sptr) (StaticPtr sptr))
+cmap sf = cap (closure sf)
+{-# DEPRECATED cmap "Use staticMap instead." #-}
```

Comment:
Or `cap` for that matter.

Which, I guess, is both `staticMap` _and_ `staticApply`.